src/components/AddPeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -55,13 +55,13 @@
           </v-tooltip>
         </v-col>
         <v-col>
-          <base-address class="peoples-roles-mailing-address" :address="person.address.mailingAddress" />
+          <base-address class="peoples-roles-mailing-address" :address="person.mailingAddress" />
```

Comment:
To reduce confusion about object names, I'd like to see the iterator here be changed from 'person' to 'officer'. Would that make sense?

(This is so you'd always be using `officer.xxx` instead of sometimes person.xxx and other times officer.xxx.)

Reply:
> A few comments but looks otherwise OK.

Hey! thanks for the review, sorry i'm going to pull back this PR as there is still significant more refactoring to come based on the changes from Friday's schema change. I'll post up another and tag for re-review then :) Thanks!

Reply:
Agreed, refactored!